Language/Functions/Advanced IO/pulseIn.adoc

```diff
@@ -31,9 +31,9 @@ La temporizzazione di questa funzione è stata determinata empiricamente e proba
 
 [float]
 === Parametri
-`pin`: il numero del pin dal quale  vuoi leggere l'impulso. (int)
+`pin`: il numero del pin dal quale vuoi leggere l'impulso. (int)
 
-`valore`: il tipo di impulso da leggere: può essere link:../../../variables/constants/constants/[HIGH] oppure link:../../../variables/constants/constants/[LOW]. (int)
+`valore`: il tipo di impulso da leggere: può essere link:https://www.arduino.cc/reference/it/language/variables/constants/constants/[HIGH] oppure link:https://www.arduino.cc/reference/en/language/variables/constants/constants/[LOW]. (int)
```

Comment:
The links are correct as they are. The content is rendered differently on arduino.cc than it is here on GitHub. GitHub is only used for development of these files. They are intended to be read only on arduino.cc so we do not make any accommodations for the way it is rendered on GitHub.

Verify the links are functional on the reference page on arduino.cc:

https://www.arduino.cc/reference/it/language/functions/advanced-io/pulsein

Relative links are important because it allows the reference to be moved within the arduino.cc website without having to update hundreds of links.

It is a bit confusing because here on GitHub each reference page is an AsciiDoc file, but on arduino.cc they are a folder containing an HTML file named `index.html`. The links are pointing to the folder because the file is loaded implicitly. This is why you will find the links contain an extra path level compared to what you see here on GitHub:

https://www.arduino.cc/reference/it/language/variables/constants/constants/index.html